test/e2e-tests-upgrade.sh

```diff
@@ -66,17 +66,29 @@ uninstall_pipeline_crd_version $PREVIOUS_PIPELINE_VERSION
 header "Install the previous release of Tekton pipeline $PREVIOUS_PIPELINE_VERSION"
 install_pipeline_crd_version $PREVIOUS_PIPELINE_VERSION
 
+header "Create resources at previous release version"
+kubectl create namespace upgrade
+kubectl create -f ./test/upgrade/simpleResources.yaml
```

Comment:
can you use `trap` to clean up the resources created during test execution?

Reply:
Thanks, I used trap to clean up the namespace.